alf/data_structures.py

```diff
@@ -110,6 +111,37 @@ def is_last(self):
         raise ValueError('step_type is not a Torch Tensor')
 
 
+def _create_timestep(observation, prev_action, reward, discount, env_id,
```

Comment:
If this function is only used for testing, then I guess it shouldn't belong to data_structures.py. Otherwise it should be moved to @runjerry 's implementation.

Reply:
Jerry is just using constructor for now.  Probably can be merged.  I'll leave it up to @runjerry?

Reply:
I am not using it right now, but will leave it after the re-implementation.